ntbs-service/Pages/ClinicalTimelines/Edit.cshtml

```diff
@@ -12,8 +14,9 @@
 
 <div class="row">
     <date-comparison inline-template>
-        <div class="col-md-5">
-            <form method="post">
+        <!-- TODO style="width: 100%" is needed currently (possibly due to the use of col-md-5 div)  -->

```

Comment:
The <form> and <div> tags have been swapped as the pagination is now included on this page - I've left the styling inline for width: 100% as I expect this isn't final, happy to make a class for it though if we don't want anything like this left in the code just let me know.